composables/useToggleApprovall.js

```diff
@@ -11,7 +11,7 @@ export const useToggleApproval = ({ eula }) => {
   const createBody = (value) => ({
```

Comment:
**todo:** I just noticed the typo in the file name. It should be `useToggleApproval` with just one `l`, like the function itself.

Reply:
Done!